content/en/docs/armory-agent/agent-mtls.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Configure Mutual TLS Authentication"
+linkTitle: "Configure mTLS"
+weight: 40
+description: >
+  Configure mTLS authentication in the Armory Agent plugin and service.
+---
+
+## {{% heading "prereq" %}}
+
+You need the following to configure mTLS:
+
+* `CA.crt`
+* Clouddriver certificate and private key
+* Agent certificate and private key
+
+For testing, you can create self-signed certificates and keys like this:
+
+```bash
+# The first step is to create a certificate authority (CA) that both the Agent and Clouddriver trust
+openssl req -new -x509 -nodes -days 365 -subj '/CN=my-ca' -keyout ca.key \
+    -out ca.crt
+
+openssl x509 -in ca.crt -text -noout
+
+#Clouddriver cert
+openssl genrsa -out clouddriver.key 2048
+
+openssl req -new -key clouddriver.key -subj '/CN=localhost' -out clouddriver.csr
+
+openssl x509 -req -in clouddriver.csr -CA ca.crt -CAkey ca.key -CAcreateserial \
+    -days 365 -out clouddriver.crt
+
+openssl x509 -in clouddriver.crt -text -noout
+
+#agent certificate
+openssl genrsa -out agent.key 2048
+
+openssl req -new -key agent.key -subj '/CN=my-client' -out agent.csr
+
+openssl x509 -req -in agent.csr -CA ca.crt -CAkey ca.key -CAcreateserial \
+    -days 365 -out agent.crt
+```
+
+
+
+## Agent plugin configuration
+
+### Create a Kubernetes secret
+
+Create a secret that contains `clouddriver.crt` and `clouddriver.pem`.
```

Comment:
Right below this is a .key file and no .pem file. The .crt is in pem format, I believe, so this should read "and the corresponding key"